fearless/Modules/WalletDetails/WalletDetailsInteractor.swift

```diff
@@ -23,6 +26,34 @@ extension WalletDetailsInteractor: WalletDetailsInteractorInputProtocol {
     func setup() {
         fetchChainsWithAccounts()
     }
+
+    func update(_ walletName: String) {
```

Comment:
I think param name is required there, otherwise context may be missed, because interactor can update not only walletName

Reply:
Agree, I changed function name several times and missed it